geographic_msgs/include/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+target_include_directories(validation
+  INTERFACE
```

Comment:
Could this one-liner not just be in the make CMakelists?

Reply:
It's a 4-liner, but it could be in either one. It doesn't matter.